internal/repository/root.go

```diff
@@ -100,12 +123,18 @@ func (r *Repository) AddTopLevelTargetsKey(ctx context.Context, rootKeyBytes, ta
 	if err != nil {
 		return err
 	}
+	slog.Debug("Signed DSSE envelope via signer verifier")
```

Comment:
This can and should be consistent for each function, the signing operation is the same whether it's when initializing root or updating it.